index.css

```diff
@@ -701,3 +701,406 @@ body {
   margin-right: 20px;
 }
 
+@media screen and (min-width: 769px) {
```

Comment:
Your contact-me section layout is also not as per the Figma template. Please use the same margins, and padding and follow the layout of the Figma template.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177910261-a91eaab1-fc89-48f3-b15a-5ae8f8848198.png) | ![image](https://user-images.githubusercontent.com/33268927/177910306-7f8f813a-78dc-4bb2-bb04-00770c140b99.png) |